playwright/_impl/_transport.py

```diff
@@ -23,6 +23,7 @@
 from typing import Callable, Dict, Optional, Union
 
 import websockets
+import websockets.exceptions
```

Comment:
how did this result in a linting error? since the CI was green I'm confused.

Reply:
Try hovering over websockets.exceptions it is missing import which is allowed in the mypy config, with this you will get intellisense :)